ui/src/app/views/cohort-builder-placeholder/component.css

```diff
@@ -0,0 +1,4 @@
+img {
+  cursor: pointer;
+  cursor: hand;
```

Comment:
Can you add `max-width: 100%`? This runs off my screen:

![image](https://user-images.githubusercontent.com/313089/28481254-2ef21f54-6e32-11e7-9df2-ad5c1735ee98.png)